### ISSUE TITLE
EPIC: As Open Data Hub manager I would like to integrate flight services in the v2 OTP instance

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/opendatahub-mentor-otp#170]
GTFS end-point flight data: https://gtfs.api.opendatahub.com/v1/dataset/skyalps-flight-data/raw

GTFS-RT end-points:
https://mobility.api.opendatahub.testingmachine.eu/v2/GTFS-RT
https://mobility.api.opendatahub.testingmachine.eu/v2/GTFS-RT-JSON

**Comment by leonardehrenfried** [comment]
I tested this today and it doesn't appear to be possible to select the airports outside of Alto Adige in the newly deployed UI, you can request them in the API: https://tinyurl.com/275wsuld

**Comment by rcavaliere** [comment]
@leonardehrenfried yes it should be like this, at present we just have the flight arriving or departing in Bolzano Airport

**Comment by rcavaliere** [comment]
@leonardehrenfried were you able to see some flights on the UI connected to the new OTP instance which has also integrated the flight data, i.e. https://otp.opendatahub.testingmachine.eu? I was not able to get some itineraries suggested.
In the meantime the number of flights has increased, and we have all the flights of this operator around Europe!
You can check this there: https://webcomponents.opendatahub.com/webcomponent/flightdata-realtime

**Comment by leonardehrenfried** [comment]
No, not with the UI because there we only have street (OSM) coverage in South Tyrol. We would have to teach the UI to use the id of the airport as the destination not coordinates near the airport.

It does work in the API, though: https://tinyurl.com/23wlcxx4
